tools/Equinox.Tool/Program.fs

```diff
@@ -179,13 +179,14 @@ and [<NoComparison; NoEquality>] TestParameters =
     | [<AltCommandLine "-d">]               DurationM of float
     | [<AltCommandLine "-e">]               ErrorCutoff of int64
     | [<AltCommandLine "-i">]               ReportIntervalS of int
-    | [<CliPrefix(CliPrefix.None); Last>]                      Cosmos   of ParseResults<Storage.Cosmos.Parameters>
-    | [<CliPrefix(CliPrefix.None); Last>]                      Dynamo   of ParseResults<Storage.Dynamo.Parameters>
-    | [<CliPrefix(CliPrefix.None); Last>]                      Es       of ParseResults<Storage.EventStore.Parameters>
-    | [<CliPrefix(CliPrefix.None); Last>]                      Memory   of ParseResults<Storage.MemoryStore.Parameters>
-    | [<CliPrefix(CliPrefix.None); Last; AltCommandLine "ms">] MsSql    of ParseResults<Storage.Sql.Ms.Parameters>
-    | [<CliPrefix(CliPrefix.None); Last; AltCommandLine "my">] MySql    of ParseResults<Storage.Sql.My.Parameters>
-    | [<CliPrefix(CliPrefix.None); Last; AltCommandLine "pg">] Postgres of ParseResults<Storage.Sql.Pg.Parameters>
+    | [<CliPrefix(CliPrefix.None); Last>]                         Cosmos    of ParseResults<Storage.Cosmos.Parameters>
+    | [<CliPrefix(CliPrefix.None); Last>]                         Dynamo    of ParseResults<Storage.Dynamo.Parameters>
+    | [<CliPrefix(CliPrefix.None); Last>]                         Es        of ParseResults<Storage.EventStore.Parameters>
+    | [<CliPrefix(CliPrefix.None); Last>]                         Memory    of ParseResults<Storage.MemoryStore.Parameters>
+    | [<CliPrefix(CliPrefix.None); Last; AltCommandLine "ms">]    MsSql     of ParseResults<Storage.Sql.Ms.Parameters>
+    | [<CliPrefix(CliPrefix.None); Last; AltCommandLine "my">]    MySql     of ParseResults<Storage.Sql.My.Parameters>
+    | [<CliPrefix(CliPrefix.None); Last; AltCommandLine "pg">]    Postgres  of ParseResults<Storage.Sql.Pg.Parameters>
+    | [<CliPrefix(CliPrefix.None); Last; AltCommandLine "msgdb">] MessageDb of ParseResults<Storage.MessageDb.Parameters>
```

Comment:
maybe use `Mdb` as the case? All the others don't have DB and are pithy (also needs to happen in storage.fs if it happens)